recipes/r-mlpack/meta.yaml

```
@@ -0,0 +1,100 @@
+{% set version = '3.4.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-mlpack
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/mlpack_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/mlpack/mlpack_{{ version }}.tar.gz
+  sha256: 3d2271769972309ada484ae3a3346b8854269ff6b65ba163d575498a04b2de8f
+
+build:
+  merge_build_host: True  # [win]
+  skip: true  # [r_base == "3.6"]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - llvm-openmp                  # [osx]
+    - r-base
+    - r-bh >=1.58
+    - r-rcpp >=0.12.12
+    - r-rcpparmadillo >=0.8.400.0
+    - r-rcppensmallen >=0.2.10.0
+    - libblas
+    - libgomp      # [linux]
+    - liblapack
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-bh >=1.58
```

Not required as header only.
```suggestion
    #- r-bh >=1.58
```